packages/cli/templates/element.config.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
 		devtools: false,
 		sandbox: true,
 		watch: false,
-		stepDelay: 0,
-		actionDelay: 0,
+		stepDelay: '0s',
+		actionDelay: '0s',
```

Comment:
'0' is ok I think lol
```suggestion
		stepDelay: '0',
		actionDelay: '0',
```